src/main/java/uk/nhs/digital/ps/migrator/model/hippo/NationalIndicatorMigrator.java

```diff
@@ -50,20 +48,10 @@ public NationalIndicatorMigrator(ExecutionParameters executionParameters) {
 
         int titleCol = titleCols.get(0);
 
-        final NationalIndicatorFolder rootFolder = new NationalIndicatorFolder(
-            null,
-            ROOT_CATALOG_LABEL
-        );
-
-        rootFolder.setLocalizedName(ROOT_CATALOG_LABEL);
-        rootFolder.setJcrNodeName("national-indicator-library");
-
-        indicators.add(rootFolder);
-
```

Comment:
I shoul've thought about it when fixing this, but would would you mind adding some comment in place of this removed block, explaining that it's expected that the 'National Indicator Library' root folder already exists?

Just to spare someone else our yesterday's grief :wink: